src/azure-cli/azure/cli/command_modules/aro/_params.py

```diff
@@ -62,44 +62,47 @@ def load_arguments(self, _):
                    validator=validate_version_format)
 
         c.argument('pod_cidr',
-                   help='CIDR of pod network. Must be a minimum of /18 or larger.',
+                   help='CIDR of pod network. Must be a minimum of /18 or larger. [Default: 10.128.0.0/14]',
```

Comment:
Please don't add default value in the argument help. You should set the default value in custom function signature, and the framework will add them in the help message. Like this https://github.com/Azure/azure-cli/blob/964aec74efe1fe24e5b16c377f7bcb78316bb8df/src/azure-cli/azure/cli/command_modules/container/custom.py#L74-L84